No typescript definition file in 5.1.0

I get an error in my typescript project. 

> error TS7016: Could not find a declaration file for module 'express-restify-mongoose'.

I have the latest package at this time 5.1.0 but there is no type definition file in 
 my `node_modules/express-restify-mongoose`.
Also `npm install -D @types/express-restify-mongoose does` not work. 

> ERR! 404  '@types/express-restify-mongoose@latest' is not in the npm registry.

I see the `index.d.ts` file in the code directory of this repository and if I add it manually to the `node_modules/express-restify-mongoose` folder then everything works.

Please make the type definitions file part of the distribution.

--- a/index.d.ts
+++ b/index.d.ts
@@ -41,5 +41,5 @@ declare module 'express-restify-mongoose' {
 
     export function serve(router: express.Router, mongooseModel: mongoose.Model<any>, options?: Options): string;
 
-    export function defaults(options: Options);
+    export function defaults(options: Options): void;
 }

--- a/package.json
+++ b/package.json
@@ -77,6 +77,7 @@
     "test-cov-restify": "istanbul cover node_modules/mocha/bin/_mocha -- --require babel-register -R spec ./test/restify.js --timeout 10s",
     "report-coverage": "cat ./coverage/lcov.info | coveralls"
   },
+  "types": "./index.d.ts",
   "files": [
     "CHANGELOG.md",
     "LICENSE",

